docs/index.md

```diff
@@ -132,6 +134,33 @@ Run the following command to report each incident:
     --timestampResolved "<timestampResolved>"
 ```
 
+### Events Helpers
+
+In order to simplify the integration process we have some helpers that you can use to push multiple events simultaneously:
+
+-   [Deployment + Changes (Git)](#deployment--changes-git)
+
+#### Deployment + Changes (Git)
```

Comment:
I'm figuring out if, in practice, this new command is:

- An alternative to reporting **deployments**, that makes it much easier to report the included changes; or
- An alternative to reporting both **changes** and **deployments** at once, meaning that customers would no longer need to report changes as they happen.

I'm clarifying this with @DReigada. :bow: